065_olm.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Variables to set, suit to your installation
+cd /root
+export PATH=/root/bin:$PATH
+export OCP_RELEASE="{{ disconnected_operators_version|default(openshift_version|default(4.7)) }}"
+export OCP_PULLSECRET_AUTHFILE='/root/openshift_pull.json'
+IP=$(hostname -I | awk -F' ' '{print $2}')
+REVERSE_NAME=$(dig -x $IP +short | sed 's/\.[^\.]*$//')
+echo $IP | grep -q ':' && SERVER6=$(grep : /etc/resolv.conf | grep -v fe80 | cut -d" " -f2) && REVERSE_NAME=$(dig -6x $IP +short @$SERVER6 | sed 's/\.[^\.]*$//')
+REGISTRY_NAME=${REVERSE_NAME:-$(hostname -f)}
+export LOCAL_REGISTRY=$REGISTRY_NAME:5000
```

Comment:
so it have to be 5000?

Reply:
also LOCAL here is misleading, actually ppl can run the registry externally to the installer, right (as long as they provide the access info)

Reply:
the plan always uses local registry

Reply:
yeah, I'd say, that's a missing feature (to support external disconnected registry)